pub_sub/java/http/checkout/src/main/java/com/service/CheckoutServiceApplication.java

```diff
@@ -0,0 +1,52 @@
+package com.service;
+
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+public class CheckoutServiceApplication {
+	private static final HttpClient httpClient = HttpClient.newBuilder()
+			.version(HttpClient.Version.HTTP_2)
+			.connectTimeout(Duration.ofSeconds(10))
+			.build();
+
+	private static final String PUBSUB_NAME = "order_pub_sub";
+	private static final String TOPIC = "orders";
+	private static String DAPR_HOST;
+	private static String DAPR_HTTP_PORT;
+
+	public static void main(String[] args) throws InterruptedException, IOException {
+		String daprHost = System.getenv("DAPR_HOST");
```

Comment:
Consider this for optional env vars with default values

` String DAPR_HOST= System.getenv().getOrDefault("DAPR_HOST", "http://localhost"); `
` String DAPR_HTTP_PORT= System.getenv().getOrDefault("DAPR_HTTP_PORT", "3500"); `